apps/frontend/src/renderer/components/gitlab-merge-requests/components/SeverityGroupHeader.tsx

```
@@ -0,0 +1,72 @@
+  return (
+    <button
+      type="button"
+      onClick={onToggle}
+      className="w-full flex items-center justify-between p-3 hover:bg-black/5 dark:hover:bg-white/5 rounded-t-lg transition-colors"
+    >
+      <div className="flex items-center gap-3">
+        {/* Group Checkbox */}
+        <div
+          onClick={onSelectAll}
+          className="cursor-pointer"
+        >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)